pkg/services/ngalert/store/provisioning_store_test.go

```diff
@@ -116,6 +117,6 @@ func TestProvisioningStore(t *testing.T) {
 	})
 }
 
-func createSut(_ *ngalert.AlertNG, db *store.DBstore) (store.ProvisioningStore, store.TransactionManager) {
+func createSut(_ *ngalert.AlertNG, db *store.DBstore) (services.ProvisioningStore, services.TransactionManager) {
```

Comment:
I realized this naming is bad, because the most private a function can get in Go is still available to the entire package. So, this takes a name which many test files would potentially need.